shared/actions/team-building.tsx

```diff
@@ -115,16 +116,30 @@ const search = (state: TypedState, {payload: {namespace}}: NSAction) => {
 }
 
 const fetchUserRecs = (state: TypedState, {payload: {namespace}}: NSAction) =>
-  RPCTypes.userInterestingPeopleRpcPromise({maxUsers: 50})
-    .then((suggestions: Array<RPCTypes.InterestingPerson> | null) =>
-      (suggestions || []).map(
-        ({username, fullname}): TeamBuildingTypes.User => ({
-          id: username,
-          prettyName: fullname,
-          serviceMap: {keybase: username},
+  Promise.all([
+    RPCTypes.contactsLookupSavedContactsListRpcPromise(),
+    RPCTypes.userInterestingPeopleRpcPromise({maxUsers: 50}),
+  ])
+    .then(results => {
+      const contactUsernames = new Set(results[0].map(x => x.username).filter(Boolean))
+      const contacts = results[0].map(
```

Comment:
@buoyad maybe we should make the service do all of this work. Initial idea was to have separate RPCs but it looks we will need some logic if we go this road.

Reply:
Hm, ok, I think this is fine for now.